Tasks/VsTestV2/distributedtest.ts

```diff
@@ -112,7 +100,18 @@ export class DistributedTest {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
 
-            const sources = tl.findMatch(this.inputDataContract.TestSelectionSettings.SearchFolder, sourceFilter);
+            let sources;
+            try {
+                if (!utils.Helper.stringToBool(tl.getVariable('EnableVerboseFilesListing'))) {
```

Comment:
why we need this? Enable/Disable -> there shouldn't be another secret flag to do.